crates/bevy_tasks/src/lib.rs

```diff
@@ -34,10 +36,73 @@ pub mod prelude {
     };
 }
 
+// The following docs are copied from `num_cpus`'s docs for the wrapped functions.
```

Comment:
I suspect we should just re-export the items with a different name in that case.

I suppose that would make the examples have the wrong paths :(

I mean technically that does make it observable that we are using num_cpus, but and code relying on that would be very contrived.